misc/config_tools/requirements.txt

```diff
@@ -1,10 +1,10 @@
 defusedxml
 lxml
-elementpath
-xmlschema
+elementpath==2.5.3
+xmlschema==1.9.2
 build
 tqdm
 requests
 xmltodict
 sphinx
```

Comment:
The previous list (in the GSG) only had 5 python packages.  Why do we need the new ones added here: build, requests, xmltodict, and sphinx?  (If they're for the configurator, we don't have the user build the configurator any more so we don't need those in the general developer prerequisites.

We need an older version of sphinx for documentation and not just the latest available.